src/components/KMultiselect/KMultiselect.vue

```diff
@@ -32,44 +29,70 @@
             return
           }"
           :position-fixed="positionFixed"
-          :target="`[id='${multiselectInputId}']`"
-          :test-mode="!!testMode || undefined"
+          target=".multiselect-trigger"
           @closed="() => handleToggle(false, isToggled, toggle)"
           @opened="() => handleToggle(true, isToggled, toggle)"
         >
           <div
-            ref="multiselectRef"
-            class="k-multiselect-trigger"
+            ref="multiselectElement"
+            class="multiselect-trigger"
             :class="{ focused: isFocused, hovered: isHovered, disabled: isDisabled, readonly: isReadonly }"
-            data-testid="k-multiselect-trigger"
+            data-testid="multiselect-trigger"
             role="listbox"
             @click="handleFilterClick"
           >
+            <div v-if="collapsedContext">
+              <KInput
+                :id="multiselectId"
+                v-bind="modifiedAttrs"
+                ref="multiselectInputElement"
+                autocapitalize="off"
+                autocomplete="off"
+                class="multiselect-input"
+                :class="{ 'is-readonly': isReadonly }"
+                data-testid="multiselect-input"
+                :model-value="filterString"
+                :placeholder="getPlaceholderText"
+                :readonly="isReadonly ? true : undefined"
+                type="text"
+                @blur="() => isFocused = false"
```

Comment:
nit: I think pressing escape with the input focused should close the dropdown (not sure if this goes here or the other input)

Reply:
seems to work now 🤷🏼